### DISCUSSION TITLE
关于 alist 引用依赖的规范

### DISCUSSION THREAD

**Discussion by potoo0** [AlistGo/alist#6400] [General]
一个大型项目的依赖引入时应该要一些规范吧。

比如现在引用了 rclone，而 rclone 也是一个相当庞大复杂的工具，为了使用这个工具中几个方法而引入整个依赖是十分不推荐的

![image](https://github.com/alist-org/alist/assets/34411681/988bd340-f0c8-4dce-8072-bb2994013ab1)

**Comment by potoo0** [comment]
@xhofe

**Comment by IAALAI** [comment]
确实。。。感觉现在的包有点多了
